### PR TITLE
Update akka, swagger and scalatest

### PR DESCRIPTION
**Pull request by KaloyanTanev**
Apologies for the huge PR, turned out bigger than intended as there were multiple inter-dependant libraries. There are no new features or bugfixes only version upgrades and the necessary refactoring of the code. It may look enormous but it's the same thing replicated through multiple of files.

- upgrade akka-http module to 10.2.7
- refactor akka-http as somehow minor versions are incompatible
- upgrade swagger to v3 (which includes upgrade from OpenAPI2 to OpenAPI3), required from akka-http 10.2.7
- refactor swagger
- upgrade swagger's UI
- upgrade akka module to 2.6.16
- upgrade scalatest and scalactic to 3.1.4 and refactor the imports in the test cases as it's not backwards compatible
- add scalatestplus, because of scalatest incompatibility (check https://github.com/scalatest/scalatest/issues/1735)

P.S: you can ignore the file changes in `src/main/resources/swagger-ui`, it's just 2.X distribution package changed with 3.52.2 distribution package (https://github.com/swagger-api/swagger-ui/tree/v3.52.5/dist)

### CODE REVIEW

**Comment by jasny** [comment]
@KaloyanTanev Please check why the unit tests are failing

**Comment by KaloyanTanev** [comment]
@jasny Yap, that's what I've been doing yesterday (related to my question about the versions on DM). The akka-http has different version of scalatest, which interferes with the direct dependency of scalatest we have :(. And again, disappointed but not surprised, minor version upgrades are incompatible...

**Comment by jasny** [comment]
Exclude `scalatest` as dependency for akka-http.

**Comment by KaloyanTanev** [comment]
@jasny I've tried excluding it in both akka-http and akka, but didn't seem to work :(. The dependency tree in sbt showed it's not there, but it continued malfunctioning with the same error... Anyway, I was halfway through the update, so I just finished it, I guess it's even better that way - having newer version.

**Comment by jasny** [comment]
@KaloyanTanev Strange. Please exclude it anyway, even now it works without excluding it.

**Review by jasny — APPROVED** [review]

File: src/main/scala/com/ltonetwork/api/http/swagger/SwaggerDocService.scala [hunk 1]
```diff
@@ -23,8 +24,16 @@ class SwaggerDocService(val actorSystem: ActorSystem, val apiClasses: Set[Class[
     .description("The Web Interface to the LTO Public Node API")
     .license(license)
 
+  val scheme = new SecurityScheme()
+  scheme.setType(SecurityScheme.Type.HTTP)
+  scheme.setScheme("bearer")
+  scheme.setBearerFormat("JWT")
```
[the remark this answers is not in the document]
Comment (KaloyanTanev) [reply src/main/scala/com/ltonetwork/api/http/swagger/SwaggerDocService.scala:30]: Ah, I thought it is... my bad, thanks!

**Pull request merged**